### PR TITLE
added show for bookings

### PR DESCRIPTION
**Pull request by danpring**
added show method for bookings and updated pundit authorisations for new methods.

### CODE REVIEW

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -14,7 +16,14 @@ def create
     @booking.status = "pending"
     binding.pry
```
Comment (JulianS61) [inline app/controllers/bookings_controller.rb:17]: TODO: delete binding.pry

File: app/policies/booking_policy.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+  end
+
+    def new?
+      true
+    end
+
```
Comment (JulianS61) [inline app/policies/booking_policy.rb:11]: Don't need the new method since it inherits create value

File: app/policies/equipment_policy.rb [hunk 3]
```diff
@@ -5,7 +5,19 @@ def resolve
     end
   end
 
+  def new?
+    true
+  end
+
```
Comment (JulianS61) [inline app/policies/equipment_policy.rb:11]: the same

File: app/controllers/bookings_controller.rb (hunk 1 above)
Comment (tieuu) [reply app/controllers/bookings_controller.rb:17]: oh yeah

**Review by JulianS61 — CHANGES_REQUESTED** [review]
seems ok, except some improvement

**Comment by JulianS61** [comment]
Since it doesn't create any bug, i'll just merge. And someone could correct it on another commit.

**Pull request merged**